### DISCUSSION TITLE
Problems with email delivery to Outlook

### DISCUSSION THREAD

**Discussion by DigitalGoldfish** [epicweb-dev/epic-stack#409] [General]
I am having problems with email deliverability to Outlook mailboxes using resend. They are listed as delivered in the resend admin UI but  I don't receive anything in my mailbox. Sending to other mailboxes (Gmail etc.) works fine. Resend support staff isn't helpful just pointing to the delivered status in the backend. 

Anyone have similar experiences?

**Comment by MoSattler** [comment]
Hey @DigitalGoldfish,

Sorry you're facing this with resend to Outlook. I am not resend expert, but a few quick thoughts:

1. **Spam Folders:** Have you checked if emails are landing in Outlook's junk?
1. **Email Configuration:** Are your DNS records set right for your resend domain?
1. **Test Content:** Did you try sending a simple email without much formatting and little content? Just to see if you trigger some some sort of spam filter.
1. **Email Headers:** Can you pull the headers from the 'delivered' emails in the resend admin? (Not sure if they offer that, but maybe ask support)
1. **Outlook Issues:** Have you searched the [developer forum](https://techcommunity.microsoft.com/t5/outlook/ct-p/Outlook)?

By the way, is this issue consistent for all Outlook recipients or just your mailbox?

**Reply by DigitalGoldfish** [reply]
1) Yes I checked, nothing in my SPAM folders. But I am going to talk with the Office 365 Admin today if there is some sort of global spam/jail folder that only Admins can see. 
2) Yes according to resend
3) I will try this today
4) I will check today
5) I did, found nothing relevant.

I didn't check with everyone but I asked a colleague to verify and he didn't receive an email either.

**Comment by kentcdodds** [comment]
@DigitalGoldfish, 

I'm confused because you posted https://github.com/epicweb-dev/epic-stack/issues/410

So are you getting emails in Outlook or not?

**Reply by DigitalGoldfish** [reply]
@kentcdodds  I am not getting any emails in outlook using resend. But I switched back to mailgun after I couldn't get it working in resend and then ran into issue #410.

**Comment by DigitalGoldfish** [comment]
I've found the missing emails. They were quarantined by MS Defender as a "High confidence Phish" and moved to a quarantine folder. According to our admin we just use the default rules in MS Defender so I would assume other customers to be affected as well. 

I've also triggered a password reset email on resend.com (The only way I could think to get an email from resend.com using a different account). It too got quarantined as a "High confidence phish". I've reported them as "clean/false positive" to Microsoft but not sure if that will change anything. It might help if more people report them as false positives but not sure how many here use outlook since no one else ran into the problem. 

As a lot of our users use office 365 we will probably just use MailGun for the time being.

**Reply by MoSattler** [reply]
Interesting! You should definitely report this to resend as well - if anyone can do anything about that it's them.

**Reply by DigitalGoldfish** [reply]
I've already reported it to resend along with screenshots of everything MS Defender tells me about the mails.
